### ISSUE TITLE
binary report file dot-extension

### ISSUE THREAD

**Issue by samhatchett** [OpenWaterAnalytics/EPANET#61]
wonder if we should have a poll here - there are various dot extensions that we have used for the binary output file... this would be useful to standardize so we can write scripts that know what files to look for. some ones used previously and some suggestions:
- `.out` "output"
- `.bin` "binary"
- `.enb` "epanet binary"
- `.ebr` "epanet binary report"
- `.erd` "epanet report database"

anyone have strong opinions? I would prefer to use some extension that would distinguish itself from the _other_ binary file (used for hydraulics, with uneven time steps).

**Comment by eladsal** [comment]
Yes, we should agree on the extensions. EPANET uses 4 [files](https://github.com/OpenWaterAnalytics/EPANET/wiki/File-Descriptions): input (INP), report, output and hydraulic.

What I have seen and used are .rpt for report, .out for the binary output and .hyd for the hydraulic file. *.bin is also fine by me for the binary output.

**Comment by LRossman** [comment]
@samhatchett ,
We are trying to clean up this project's list of open issues by closing those that have either been addressed or are no longer relevant. You can re-open this issue if you wish to keep it active.
